### PR TITLE
Test swagger docs.

### PR DESCRIPTION
**Pull request by fisx**
## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: integration/test/Testlib/HTTP.hs [hunk 1]
```diff
@@ -101,6 +102,9 @@ onFailureAddResponse r m = App $ do
 
 data Versioned = Versioned | Unversioned | ExplicitVersion Int
 
+localRawBaseRequest :: HasCallStack => Service -> Versioned -> String -> App HTTP.Request
+localRawBaseRequest = rawBaseRequest OwnDomain
```
Comment (pcapriotti) [inline integration/test/Testlib/HTTP.hs:106]: What's the point of this definition? I'd rather see `rawBaseRequest OwnDomain` than `localRawBaseRequest` and have to look up what that is :shrug:.

File: integration/test/API/Brig.hs [hunk 2]
```diff
@@ -212,3 +212,48 @@ putUserSupportedProtocols user ps = do
...
+
+getSwaggerPublicTOC :: HasCallStack => App Response
+getSwaggerPublicTOC = do
+  req <-
+    localRawBaseRequest Brig Unversioned $
+      joinHttpPath ["api", "swagger-ui"]
+  submit "GET" req
+
+getSwaggerInternalTOC :: HasCallStack => App Response
+getSwaggerInternalTOC = error "FUTUREWORK: this API end-point does not exist."
```
Comment (pcapriotti) [inline integration/test/API/Brig.hs:231]: Doesn't this simply crash? Maybe turn it into a comment.

**Review by pcapriotti — APPROVED** [review]

File: integration/test/API/Brig.hs (hunk 2 above)
Comment (fisx) [reply integration/test/API/Brig.hs:231]: it's meant to crash.  nobody should call this function before it is doing anything interesting.  also, crashing isn't that bad in a test suite.  just stop calling this function, to resolve it?

File: integration/test/Testlib/HTTP.hs (hunk 1 above)
Comment (fisx) [reply integration/test/Testlib/HTTP.hs:106]: the idea was that somebody who doesn't know about `OwnDomain` can find this function and will understand that it's the thing they were looking for.  fewer questions from stupid developers you'll have to answer in the future.  :)

but no strong opinion, i'll remove it.  :+1:

File: integration/test/API/Brig.hs (hunk 2 above)
Comment (pcapriotti) [reply integration/test/API/Brig.hs:231]: You're right, sorry, I thought this was a test. My mistake.

**Pull request merged**